stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/QuotedMessageText.kt

```diff
@@ -83,7 +85,12 @@ public fun QuotedMessageText(
         "quotedMessageText is null. Cannot display invalid message title."
     }
 
-    val styledText = buildAnnotatedMessageText(quotedMessageText, message.isMine(ChatClient.instance()))
+    val textColor = if (replyMessage?.isMine(ChatClient.instance()) != false) {
```

Comment:
Same question here about the possibility of safely flipping the boolean evaluation.